src/AltinnCore/RuntimeDataService/Controllers/ReporteeElementController.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AltinnCore.Runtime.DataService.Models;
+using AltinnCore.Runtime.DataService.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace AltinnCore.Runtime.DataService.Controllers
```

Comment:
I dont think we should use Runtime in the namespace

Reply:
Fixed